src/components/panel/TextViewsToggle.tsx

```diff
@@ -50,6 +51,11 @@ const TextViewsToggle: FC = () => {
     updatePanel(panelState.id, { view: newView })
   }
 
+  function isEnabled(buttonView: ViewType) {
+    return !(!panelState.imageExists && ['image', 'swap', 'split'].includes(buttonView))
```

Comment:
Hm this is less helpful than I thought, maybe change it back to your original condition and name the function ìsDisabled`